libs/common/radix_tree.hpp

```diff
@@ -10,6 +10,7 @@
 #include <memory>
 #include <string>
 #include <vector>
+#include <optional>
```

Comment:
use ```clang-format-7``` plz

Reply:
It moved `#include <optional> ` after `#include <memory>`.
___________
There are other changes 13 changes of formatting after running the command -I've ignored its as not connected with the issue.